patches/components-version_ui-resources-about_version.html.patch

```diff
@@ -0,0 +1,15 @@
+diff --git a/components/version_ui/resources/about_version.html b/components/version_ui/resources/about_version.html
+index a6ddaff346ca5fafcc6f4156da607a997af20349..eff226702323dff79f6150c63f62e54b3ab5667d 100644
+--- a/components/version_ui/resources/about_version.html
++++ b/components/version_ui/resources/about_version.html
+@@ -42,6 +42,10 @@ about:version template page
+ </if>
+         <div id="company">$i18n{company}</div>
+         <div id="copyright">$i18n{copyright}</div>
++        <hr>
```

Comment:
I don't think we need to add anything here for an internal ui used for debugging. That just doesn't seem like it would justify a patch

Reply:
See https://github.com/brave/brave-browser/issues/1174#issuecomment-445906938 and other comments in that issue for how this link came to be. Totally happy to move it, but it seemed like this was the only choice that seemed to satisfy everyone while also minimizing patching.

Reply:
@petemill if we are forced to keep this is there a better way to do it?

Reply:
Just so this is documented here: We're going to remove the link from chrome://version and brave/brave-browser#3538 was entered to track where it should ultimately end up.